.changeset/olive-points-shave.md

```diff
@@ -0,0 +1,8 @@
+---
+'@twilio-paste/badge': major
+'@twilio-paste/core': major
+'@twilio-paste/box': minor
```

Comment:
Ah, this may not be immediate obvious. 

We actually need a changeset per component, unless you don't mind the same entry in the changelogs of all components in the changeset. 

So for example, I wouldn't want to read about badge changes in the Box changelog, so box should have it's own changelog, which also applies to core.

Reply:
Ahhh okey, that makes sense. So then I will run the changeset command 3 times?

Reply:
No just twice.

badge and core

box and core.